static/main.js

```diff
@@ -65,40 +64,39 @@ socket.on("nothing_found_on_page", function(message) {
 socket.on("init", function(message) {
     let console_div = document.getElementById("console");
     for (let table_ind = 0; table_ind < message["page_cnt"]; ++table_ind) {
-        let header = htmlToElements('<div class="row header align-items-center border" style="height: 10%">\n' +
-            '                <div class="col-lg-1 col-md-1 col-sm-1 col-1"><div class="p-0 page_index">' + (table_ind + 1).toString() + '</div></div>\n' +
-            '                <div class="col-lg-7 col-md-7 col-sm-5 col-5 align-items-center"><div class="p-3 slider_border" style="background: #111; background-clip: content-box; border-radius: 5px">\n' +
-            '                </div></div>\n' +
-            '                <div class="col-lg-3 col-md-3 col-sm-4 col-4"><div class="p-2 download_div">' +
-            '                    <div class="button-placeholder centered"><div class="button_text">Download</div></div>' +
-            '                </div></div>\n' +
-            '                <div class="col-lg-1 col-md-1 col-sm-2 col-2">\n' +
-            '                    <div class="p-1 expand_elem"><input class="dropdown" type="image" src="static/expand.png" alt="Input"> </div>\n' +
-            '                </div>\n' +
-            '            </div>\n' +
-            '\n' +
-            '            <div class="container main_body">\n' +
-            '                <div class="loading">Loading image...</div>\n' +
-            '                <div class="col-lg-5 image">\n' +
-            '                    <div class="image_div"></div>\n' +
-            '                </div>\n' +
-            '                <div class="col-lg-5 image">\n' +
-            '                    <div class="image_div_table"></div>\n' +
-            '                </div>\n' +
-            '            </div>');
-
-        let div = document.createElement("div");
-        div.id = table_ind.toString();
-        div.style.height = "40%"
-
-        for (let i = 0; i < header.length; i++) {
-            div.appendChild(header[i])
-        }
-
-        console_div.append(div);
-
-        let target_dropdown = div.querySelector(".main_body");
-        target_dropdown.style.display = "none";
+            let header = htmlToElements('<div class="row header align-items-center border" style="height: 10%">\n' +
```

Comment:
How did this happen? Indentation is wrong here, double indentation in for loop. This whole paragraph just moved one indentation to the right. I ask you one more time to check git diff to make sure that things like this do not get commited